bundles/core/src/test/java/com/adobe/cq/commerce/core/components/internal/models/v1/product/ProductImplTest.java

```diff
@@ -142,6 +139,11 @@ public void setUp() throws Exception {
 
         // context.request().adaptTo(ProductImpl.class); is moved to each test because it uses an internal cache
         // and we want to override the "slug" in testEditModePlaceholderData()
+
+        context.registerAdapter(Resource.class, ComponentsConfiguration.class, (Function<Resource, ComponentsConfiguration>) input -> input
+            .getValueMap().get("cq:graphqlClient", String.class) != null ? MOCK_CONFIGURATION_OBJECT : ComponentsConfiguration.EMPTY);
+
+        context.addModelsForPackage("com.adobe.cq.commerce");
```

Comment:
Is this really needed?

Reply:
My unit tests fail without it. I'll check again, maybe there was some other reasons for failures.